docs/en/ingest-management/elastic-agent/configuration/inputs/input-winlog.asciidoc

```diff
@@ -0,0 +1,372 @@
+:input-type: winlog
+
+:vista_and_newer: This option is only available on operating systems +
+  supporting the Windows Event Log API (Microsoft Windows Vista and newer).
+
+[[winlog-input]]
+== Winlog input
+
+++++
+<titleabbrev>Winlog</titleabbrev>
+++++
+
+Configure the `winlog` input to read messages from Microsoft Windows
+Event Channels.
+
+To configure this input, specify a `name` of the Microsof Windows
+Event Channel to subscribe to.
+
+You can get a list of available event logs by running `Get-WinEvent
+-ListLog` in PowerShell.  Most channels are tied to specific event
+publishers. Here is an example showing how to list all channels using
+PowerShell.
+
+[source,sh]
+--------------------------------------------------------------------------------
+PS C:\> Get-WinEvent -ListLog * | Format-List -Property LogName
+LogName : Application
+LogName : HardwareEvents
+LogName : Internet Explorer
+LogName : Key Management Service
+LogName : Security
+LogName : System
+LogName : Windows PowerShell
+LogName : ForwardedEvents
+LogName : Microsoft-Management-UI/Admin
+LogName : Microsoft-Rdms-UI/Admin
+LogName : Microsoft-Rdms-UI/Operational
+LogName : Microsoft-Windows-Windows Firewall With Advanced Security/Firewall
+...
+--------------------------------------------------------------------------------
+
+You must specify the full name of the channel in the configuration
+file.  To read events from an archived `.evtx` file you can specify
+the `name` as the absolute path (it cannot be relative) to the file.
```

Comment:
What does it take to read them? @kvch is working on adding 'parsers' to the new filestream input. Plus the new code structure might allow us to be more flexible in how we set up harvesters and prospectors (allow alternative implementations in the future). 

Although reading `*evtx` is still supported by this input, I wonder if we should not document that feature and think if/how we to change how this file format is supported in the future. Using the filesteam input we might combine these features such that users can configure a directory they can drop evtx files into and we would just pick those up and process them like regular log files. I feel with fleet, setups might be a little more rigid (due to users having to update configurations in Kibana) and we should consider how we can let users create a little more flexible configurations upfront.